cmake/CMakeLists_googletest_download.txt.in

```diff
@@ -15,15 +15,18 @@ if(MSVC)
   set(FORCE_SHARED_CRT "-DFORCE_SHARED_CRT=OFF")
 endif()
 
-set(OSX_RPATH "")
-if(${CMAKE_SYSTEM_NAME} MATCHES "Darwin")
-  set(OSX_RPATH "-DCMAKE_MACOSX_RPATH=1")
-endif()
-
 ExternalProject_Add(googletest
   GIT_REPOSITORY    https://github.com/google/googletest.git
   GIT_TAG           master
   SOURCE_DIR        "${GOOGLETEST_BUILD_ROOT}/googletest-src"
   BINARY_DIR        "${GOOGLETEST_BUILD_ROOT}/googletest-build"
-  CMAKE_ARGS        "-DCMAKE_MAKE_PROGRAM=${CMAKE_MAKE_PROGRAM}" "-DCMAKE_INSTALL_PREFIX=${GOOGLETEST_BUILD_ROOT}/googletest-install" "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}" "-Dgtest_force_shared_crt=ON" "-DBUILD_SHARED_LIBS=1" "${OSX_RPATH}" "-DCMAKE_POSITION_INDEPENDENT_CODE=ON" "${FORCE_SHARED_CRT}"
+  CMAKE_ARGS
+                    "-DCMAKE_MAKE_PROGRAM=${CMAKE_MAKE_PROGRAM}"
+                    "-DCMAKE_INSTALL_PREFIX=${GOOGLETEST_BUILD_ROOT}/googletest-install"
+                    "-DCMAKE_BUILD_TYPE=${CMAKE_BUILD_TYPE}"
+                    "-Dgtest_force_shared_crt=ON"
+                    "-DBUILD_SHARED_LIBS=ON"
+                    "-DCMAKE_MACOSX_RPATH=ON"
```

Comment:
Got rid of the OS_RPATH, and added MAKE_MACOSX_RPATH directly to the CMAKE_ARGS. Something with spaces or quotes was causing a problem with ExternalProject. It should be Ok for now.